test-runner/src/app.rs

```diff
@@ -39,27 +40,26 @@ pub fn find_traces() -> Result<Vec<AppTrace>, Error> {
         Path::new(r"/var/log/mullvad-vpn/"),
         Path::new(r"/var/cache/mullvad-vpn/"),
         Path::new(r"/opt/Mullvad VPN/"),
-
```

Comment:
One thing I didn't notice before: I don't like how the newlines are removed here -- they're there for a reason. I think we need to tweak `rustfmt.toml`.

Reply:
But this isn't important enough to block this PR.